docker/Dockerfile.amd

```diff
@@ -1,18 +1,36 @@
-##############################################################
-# This Dockerfile contains AMD compilers
-##############################################################
+####################################################################################
+# This Dockerfile contains the AMD ROCm SDK as well as other useful tools for Devito
```

Comment:
as well as AMD profiling tools (Omnitrace, Omniperf) useful to Devito... or something like this...